docs/en/base/lists.md

```diff
@@ -6,21 +6,32 @@ title: Lists
 
 ## Ordered list
 
+Use an ordered list when the order of the items is important.
+
 <a href="https://ubuntudesign.github.io/vanilla-framework/examples/base/lists/ordered-list/"
     class="js-example">
     View example of the base ordered list
 </a>
 
 ## Unordered list
 
+Use an unordered list when the order of the items isn't important.
+
 <a href="https://ubuntudesign.github.io/vanilla-framework/examples/base/lists/unordered-list/"
     class="js-example">
     View example of the base unordered list
 </a>
 
-## Definition list
+## Description list
```

Comment:
Interesting, it seems there is no set name for them.

Some call them definition lists:
- https://www.w3.org/TR/html401/struct/lists.html#h-10.3

Some call them description lists:
- https://www.w3.org/TR/html5/grouping-content.html#the-dl-element
- https://html.spec.whatwg.org/multipage/semantics.html#the-dl-element